rts/motoko-rts/src/gc/copying.rs

```diff
@@ -79,6 +81,16 @@ pub unsafe fn copying_gc_internal<
         );
     }
 
+    if (*region0_ptr_loc).is_ptr() {
+        // Region0 is not always a pointer during GC?
```

Comment:
It is only not a pointer if it is uninitialized (not used). In that case, it is the scalar `0`, as defined by `static mut REGION_0: Value = NO_REGION` and `const NO_REGION: Value = Value::from_scalar(0)` in `region.rs`. This is analogous to the continuation table, although the design is probably not ideal (one could use a `nil` pointer as proper initialization at support this is in the GCs).